sdk/data/aztables/README.md

```diff
@@ -1,151 +1,415 @@
+    * `UpdateMode.REPLACE` will replace the existing entity with the given one, deleting any existing properties not included in the submitted entity
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Table tasks, including:
+
+* [Creating a table](#creating-a-table "Creating a table")
+* [Creating entities](#creating-entities "Creating entities")
+* [Querying entities](#querying-entities "Querying entities")
+
+
+### Creating a table
+Create a table in your account and get a `Client` to perform operations on the newly created table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+service, err := NewServiceClient("https://<my_account_name>.table.core.windows.net", cred, nil)
+handle(err)
+resp, err := service.CreateTable("myTable")
+```
+
+### Creating entities
+Create entities in the table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+service, err := NewServiceClient("https://<my_account_name>.table.core.windows.net", cred, nil)
+handle(err)
+
+myEntity := aztables.EDMEntity{
+    Entity: aztables.Entity{
+        PartitionKey: "001234",
+        RowKey: "RedMarker",
+    },
+    Properties: map[string]interface{}{
+        "Stock": 15,
+        "Price": 9.99,
+        "Comments": "great product",
+        "OnSale": true,
+        "ReducedPrice": 7.99,
+        "PurchaseDate": aztables.EDMDateTime(time.Date(2021, time.August, 21, 1, 1, 0, 0, time.UTC)),
+        "BinaryRepresentation": aztables.EDMBinary([]byte{"Bytesliceinfo"})
+    }
+}
+marshalled, err := json.Marshal(myEntity)
+handle(err)
+
+client, err := service.NewClient("myTable")
+handle(err)
+
+resp, err := client.AddEntity(context.Background(), marshalled, nil)
+handle(err)
+```
+
+### Querying entities
+Querying entities in the table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+client, err := aztables.NewClient("https://myAccountName.table.core.windows.net/myTableName", cred, nil)
+handle(err)
+
+filter := "PartitionKey eq 'markers' or RowKey eq 'id-001'"
+options := &ListEntitiesOptions{
+    Filter: &filter,
+    Select: to.StringPtr("RowKey,Value,Product,Available"),
+    Top: to.Int32Ptr(15),
+}
+
+pager := client.List(options)
+for pager.NextPage(context.Background()) {
+    resp := pager.PageResponse()
+    fmt.Printf("Received: %v entities\n", len(resp.Entities))
+
+    for _, entity := range resp.Entities {
+        var myEntity aztables.EDMEntity
+        err = json.Unmarshal(entity, &myEntity)
+        handle(err)
+
+        fmt.Printf("Received: %v, %v, %v, %v\n", myEntity.Properties["RowKey"], myEntity.Properties["Value"], myEntity.Properties["Product"], myEntity.Properties["Available"])
+    }
+}
+
+err := pager.Err()
+handle(err)
+```
+<!--
```

Comment:
Why is this comment thing here?

Reply:
Oh looks like  some of the Python stuff is lingering here.

Reply:
Good catch, I have another section commented out for when I merge samples.